application/src/sass/components/_select.scss

```diff
@@ -0,0 +1,5 @@
+.eff-select__conditional {
+    padding-left: 16px;
+    border-left: govuk-colour("grey-2") 4px solid;
+    margin: 12px 0 0 0
+}
```

Comment:
I don't think we want to be specifying these pixel values so exactly. GOV.UK does this below, we probably want to do something similar:

```
  .govuk-radios__conditional {
    @include govuk-responsive-margin(4, "bottom");
    margin-left: $conditional-margin-left;
    padding-left: $conditional-padding-left;
    border-left: $conditional-border-width solid $govuk-border-colour;

    .js-enabled &--hidden {
      display: none;
    }

    & > :last-child {
      margin-bottom: 0;
    }
  }
```

Reply:
Maybe?

Reply:
I don't think we need 

```
    .js-enabled &--hidden {
      display: none;
    }
```

as the element should combine with our existing `hidden` class fine?

Reply:
`$govuk-border-colour` is a good call.  Probably the non-pixel values too, although I don't think that's quite so important.

Reply:
I don't really know how to implement the conditional width values, and don't feel like learning it right now... feels like a fair amount of overhead to me. 
The Design System only defines these for checkboxes and radio buttons, and their size doesn't look nice for us, so I don't think we can just use theirs.

As-is this fixes a regession that we recently introduced, and puts things back as they were. That feels like enough to do for now.  It was just an unticketed quick fix of some spacing and is starting to escalate into 3 points or more.